odd_collector/adapters/snowflake/client.py

```diff
@@ -244,12 +246,23 @@ def _(column: Column) -> bool:
         with self.connect() as cursor:
             tables: List[Table] = self._fetch_tables(cursor)
             columns: List[Column] = self._fetch_columns(cursor)
+            primary_keys: Dict[Any] = self._fetch_primary_keys(cursor)
+            clustering_keys: Dict = self._get_clustering_keys(tables)
+
+            for column in columns:
+                column.is_primary_key = False
```

Comment:
You can set fields `is_primary_key`, `is_clustering_key` with default `False` value